src/cms/data/functions/loadPermissions.js

```diff
@@ -3,11 +3,11 @@
 const createApiUser = async () => {
   const authRole = await findRole("authenticated");
   const password = await strapi.admin.services.auth.hashPassword(
-    process.env.API_USER_PASSWORD
+    process.env.STRAPI_API_USER_PASSWORD
```

Comment:
We don't necessarily have to address in this PR, but I'd like to suggest using `process.env.API_USER_PASSWORD || 'api'` for this line and a similar method on several lines below where we read from env vars but do not have a default (the build will fail without them being set properly).  Perhaps move everything up to the top of the file and set global var there and reference down below so there's a central spot for administrating the env var intake.  My main reason is for local development we can use `sqlite` as a driver instead of using a postgres db, and if we set defaults, the bootstrapping will work out of the box.  The following is a diff of `database.js` that devs can use as a local alternative option (Not sure if we want to add this as a .template or .alternative file in the same dir):

```
diff --git src/cms/config/database.js src/cms/config/database.js
index 864fb3a..4d278df 100644
--- src/cms/config/database.js
+++ src/cms/config/database.js
@@ -2,25 +2,13 @@ module.exports = ({ env }) => ({
   defaultConnection: "default",
   connections: {
     default: {
-      connector: "bookshelf",
+      connector: 'bookshelf',
       settings: {
-        client: "postgres",
-        host: env("DATABASE_HOST"),
-        port: env.int("DATABASE_PORT"),
-        database: env("DATABASE_NAME"),
-        username: env("DATABASE_USERNAME"),
-        password: env("DATABASE_PASSWORD"),
-        ssl: env.bool("DATABASE_SSL"),
-        timezone: "UTC",
+        client: 'sqlite',
+        filename: env('DATABASE_FILENAME', '.tmp/data.db'),
       },
       options: {
-        pool: {
-          min: 0,
-          max: 95,
-          idleTimeoutMillis: 30000,
-          createTimeoutMillis: 30000,
-          acquireTimeoutMillis: 30000,
-        },
+        useNullAsDefault: true,
       },
     },
   },
```

Reply:
I added a `sqlite` connection and it's the default setting.  For devs that want to use postgres, you can set the `DATABASE_CLIENT=postgres` env

I've also added the API/Admin env defaults in code

https://github.com/bcgov/bcparks.ca/pull/163/commits/f0d7521944aa045c479fb3fbe4161d75444f5335